pkg/output/output.go

```diff
@@ -22,126 +22,71 @@ import (
 	types "github.com/GoogleCloudPlatform/container-structure-test/pkg/types/unversioned"
 )
 
-const (
-	RED         = "\033[0;31m"
-	GREEN       = "\033[0;32m"
-	LIGHT_GREEN = "\033[1;32m"
-	YELLOW      = "\033[1;33m"
-	ORANGE      = "\033[0;33m"
-	CYAN        = "\033[0;36m"
-	BLUE        = "\033[0;34m"
-	PURPLE      = "\033[0;35m"
-	NC          = "\033[0m" // No Color
-)
-
-// ANSI Color Escape Codes
-// Black        0;30     Dark Gray     1;30
-// Red          0;31     Light Red     1;31
-// Green        0;32     Light Green   1;32
-// Brown/Orange 0;33     Yellow        1;33
-// Blue         0;34     Light Blue    1;34
-// Purple       0;35     Light Purple  1;35
-// Cyan         0;36     Light Cyan    1;36
-
 type OutWriter struct {
 	Format  string // TODO(nkubala): implement JSON type
 	Verbose bool
 	Quiet   bool
 }
 
+func (o *OutWriter) OutputResult(result *types.TestResult) (int, int) {
```

Comment:
I dont think this needs to return anything. You don't use the return value anywhere.

Reply:
good catch, I restructured this to aggregate the totals in `FinalResults()`